src/tests/calculate.test.js

```diff
@@ -0,0 +1,15 @@
+import calculate from '../logic/calculate';
+
+test('clear display when button is clicked', () => {
+  const buttonName = 'AC';
+  const objNull = {
+    total: null,
+    next: null,
+    operation: null,
+  };
```

Comment:
- [ ] There are other functionality in the calculate.js file please make sure to test that functionality as well.
- [ ] when clicking digits (0-9),  when clicking the decimal point ('.'), when clicking the '=' button to perform a calculation, please write all the test.